### ISSUE TITLE
a bug

### ISSUE THREAD

**Issue by yyy33** [mattmc3/antidote#28]
Hi mattmc3 , If I execute some options that I set before antidote config execution, I get an error

This is option
```
setopt APPEND_HISTORY
unsetopt INC_APPEND_HISTORY
unsetopt INC_APPEND_HISTORY_TIME
setopt SHARE_HISTORY
setopt BANG_HIST
setopt HIST_VERIFY
unsetopt EXTENDED_HISTORY
unsetopt HIST_BEEP
setopt HIST_FIND_NO_DUPS
setopt HIST_IGNORE_ALL_DUPS
setopt HIST_IGNORE_DUPS
setopt HIST_SAVE_NO_DUPS
unsetopt HIST_IGNORE_SPACE 
setopt HIST_LEX_WORDS
unsetopt HIST_NO_FUNCTIONS
setopt HIST_NO_STORE
setopt HIST_REDUCE_BLANKS
setopt HIST_SAVE_BY_COPY
setopt HIST_EXPIRE_DUPS_FIRST
setopt HIST_FCNTL_LOCK
unsetopt HIST_ALLOW_CLOBBER
setopt autocd beep nomatch notify 
setopt interactivecomments 
setopt sh_word_split 
setopt GLOB_DOTS 
setopt extendedglob 

```

This is antidote config
```
_DIR=$ZDOTDIR/plugin/antidote

# clone antidote if necessary
[[ -e $_DIR ]] || git clone https://github.com/mattmc3/antidote.git $_DIR

# source antidote
. $_DIR/antidote.zsh

# generate and source plugins from ~/.zsh_plugins.txt
antidote load

unset _DIR

```

This is error
```
(eval):1: command not found: -A
(eval):1: parse error near `bundle'
(eval):1: not an identifier: [kind]
(eval):1: not an identifier: [repo]
(eval):1: parse error near `)'
antidote-bundle:62: bad math expression: operand expected at `/usr/local...'
(eval):1: command not found: -A
antidote-bundle:62: bad math expression: operand expected at `/usr/local...'
(eval):1: parse error near `bundle'
antidote-bundle:62: bad math expression: operand expected at `/usr/local...'
(eval):1: not an identifier: [kind]
antidote-bundle:62: bad math expression: operand expected at `/usr/local...'
(eval):1: not an identifier: [repo]
antidote-bundle:62: bad math expression: operand expected at `/usr/local...'
(eval):1: parse error near `)'
antidote-bundle:62: bad math expression: operand expected at `/usr/local...'
/home/yyy/.zsh/.zsh_plugins.zsh:integer:1: not an identifier: 10

```

**Comment by mattmc3** [comment]
Can you `cat ~/.zsh_plugins.txt` and include its contents here?

**Comment by mattmc3** [comment]
Nevermind. This was definitely a bug and I fixed it and will release v1.2.1 shortly. `setopt sh_word_split` caught me and I know better - I should have been using `emulate -L zsh` to have full control of the options I expect to be set in the antidote functions.
